src/3_branch.js

```diff
@@ -176,101 +178,112 @@ if (config.CORDOVA_BUILD) {
  * **Note:** `Branch.init` must be called prior to calling any other Branch functions.
  * ___
  */
-Branch.prototype['init'] = function(app_id, options, callback) {
-	if (options && typeof options == 'function') {
-		callback = options;
-		options = { isReferrable: null };
+Branch.prototype['init'] = function(key_or_id, options, callback) {
```

Comment:
let's just call the parameter branch_key -- and include in the comment that you can optionally use your app id, but it's deprecated